static/js/vue-cdr-access/tests/unit/viewType.spec.js

```diff
@@ -1,68 +1,78 @@
-import { createLocalVue, shallowMount } from '@vue/test-utils'
-import VueRouter from 'vue-router';
+import { shallowMount, flushPromises } from '@vue/test-utils'
+import { createRouter, createWebHistory } from 'vue-router';
 import viewType from '@/components/viewType.vue'
-const localVue = createLocalVue();
-localVue.use(VueRouter);
+import displayWrapper from "@/components/displayWrapper";
 
-const router = new VueRouter({
-    routes: [
-        {
-            path: '/record/uuid1234',
-            name: 'displayRecords'
-        }
-    ]
-});
-
-let wrapper, btns;
+let wrapper, btns, router;
 
 describe('viewType.vue', () => {
     beforeEach(() => {
+        router = createRouter({
+            history: createWebHistory(),
+            routes: [
+                {
+                    path: '/record/:uuid/',
+                    name: 'displayRecords',
+                    component: displayWrapper
+                }
+            ]
+        });
         sessionStorage.clear();
         wrapper = shallowMount(viewType, {
-            localVue,
-            router
+            global: {
+                plugins: [router]
+            },
+            props: {
+                browse_type: 'list-display'
```

Comment:
Does this need to be set here? Just noticing that most of the tests to start with pushing a browse_type=list-display into the url

Reply:
Nope